doc/changes/latest.rst

```diff
@@ -2,85 +2,99 @@
 
 .. include:: names.rst
 
-0.10.2.dev
-==========
+0.10.2
+======
+
+**Released September 2023**
+
+HIGHLIGHTS
+----------
+
+- :bdg-info:`Plotting` Volume plotting functions like :func:`~plotting.plot_img` now have an optional ``radiological`` parameter, defaulting to ``False``. If ``True``, this will invert the x-axis and ``L`` and ``R`` annotations to confirm to radiological conventional view. (:gh:`3172` by `Konrad Wagstyl`_ and `Yasmin Mzayek`_).
+
+- :bdg-dark:`Code` Update Decoder objects to use the more efficient ``LogisticRegressionCV`` (:gh:`3736` by `Michelle Wang`_).
+
+- :bdg-success:`API` Add ``LassoCV`` as a new estimator option for Decoder objects (:gh: `3781` by `Michelle Wang`_)
+
+- :bdg-success:`API` Add ``vmin`` and ``symmetric_cbar`` arguments to :func:`~nilearn.plotting.plot_img_on_surf` (:gh:`3873` by `Michelle Wang`_).
```

Comment:
Can highlights be just repeats of some entries of the changelog or do they have to more generic rephrasings? I think I have seen both precedents.

Also this is my very personal take of what the highlights are. Happy to go with some others.

Reply:
I think we can just use best judgement. Sometimes it makes sense to rephrase or combine related PRs into one highlight. But otherwise just copy and paste is fine IMO

Reply:
I agree on the highlights

Reply:
So we should focus on #3856 ?